### PR TITLE
Add courses page to member area

### PR DESCRIPTION
**Pull request by HalfVoxel**
- Tweak styling for youtube embeds in quizzes so that they do not end up wider than the screen on phones. Also change the color of links to make them more readable.
- Add courses page with all quizzes. If we add any quizzes in the future, they will automatically show up here.
- Add fancier shadows to the member area (looks nicer)

image

### CODE REVIEW

File: api/src/quiz/views.py [hunk 1]
```diff
@@ -115,6 +115,39 @@ def next_question(quiz_id: int, include_correct=False):
...
+    '''Returns information about all quizzes and if the given member has completed them'''
+
+    quizzes = db_session.query(Quiz).filter(Quiz.deleted_at == None).all()
+    answered_questions_per_quiz_query = (
+        db_session.query(QuizQuestion.quiz_id, func.count(func.distinct(QuizAnswer.option_id)))
+        .join(QuizAnswer, QuizQuestion.id == QuizAnswer.question_id)
+        .filter(QuizAnswer.member_id == member_id)
+        .filter(((QuizAnswer.id == None) | QuizAnswer.correct) & (QuizAnswer.deleted_at == None) & (QuizQuestion.deleted_at == None))
+        .group_by(QuizQuestion.quiz_id)
+    )
```
Comment (emanuelen5) [inline api/src/quiz/views.py:133]: Can't really wrap my head around this query, but it seems to work 😅

**Review by emanuelen5 — APPROVED** [review]
All seems in order, as usual! :)

File: public/ts/build.sh [hunk 2]
```diff
@@ -10,4 +10,5 @@ rollup /tmp/ts_output/history.js --file ../static/js/history.js --format iife -m
 rollup /tmp/ts_output/cartpage.js --file ../static/js/cartpage.js --format iife -m
 rollup /tmp/ts_output/member.js --file ../static/js/member.js --format iife -m
 rollup /tmp/ts_output/statistics.js --file ../static/js/statistics.js --format iife -m
+rollup /tmp/ts_output/courses.js --file ../static/js/courses.js --format iife -m
```
[the remark this answers is not in the document]
Comment (HalfVoxel) [reply public/ts/build.sh:13]: Prooobably not used at all. I just changed it for good measure :p

File: api/src/quiz/views.py (hunk 1 above)
Comment (HalfVoxel) [reply api/src/quiz/views.py:133]: It's kinda sketchy tbh. It counts the number of correct answers the member has submitted for each quiz. However, it assumes that a correct answer is never submitted more than once (this is guaranteed by the UI, but the database can contain multiple correct responses to a question by a single member).

Comment (HalfVoxel) [reply api/src/quiz/views.py:133]: But it's just used for statistics, so it's probably fine :p

Comment (HalfVoxel) [reply api/src/quiz/views.py:133]: No wait! I forgot I used `count(distinct(option_id))`. It should work correctly at all times. 100%. Totally. Definitely no risk of weird bugs.

**Pull request merged**